tests/foreman/api/test_hostgroup.py

```diff
@@ -191,6 +191,40 @@ def test_verify_bugzilla_1107708(self):
         self.assertEqual(len(host_attrs['all_puppetclasses']), 1)
         self.assertEqual(host_attrs['all_puppetclasses'][0]['name'], 'ntp')
 
+    @upgrade
+    @tier3
+    def test_rebuild_config(self):
+        """ 'Rebuild orchestration config' of an existing host group
+
+        :id: 58bf7015-18fc-4d25-9b64-7f2dd6dde425
+
+        :expectedresults: rebuild hostgroup orchestration configs successfully.
+
+        :CaseLevel: System
+        """
+        lce = entities.LifecycleEnvironment(organization=self.org).create()
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.publish()
+        content_view = content_view.read()
+        promote(content_view.version[0], environment_id=lce.id)
+        hostgroup = entities.HostGroup(
+            location=[self.loc],
+            organization=[self.org],
+        ).create()
+        entities.Host(
+            hostgroup=hostgroup,
+            location=self.loc,
+            organization=self.org,
+            managed=True,
+            content_facet_attributes={
+                'content_view_id': content_view.id,
+                'lifecycle_environment_id': lce.id,
+            },
+            ).create()
+        hostgroup = hostgroup.read()
```

Comment:
Do you need this line? `rebuild_config` operates on hostgroup id, which does not change after creating and assigning new host.

Reply:
ahh .. that is not required actually ... thanks to highlight it .
push a new commit to remove same.